docs/UserGuide.md

```diff
@@ -55,12 +55,12 @@ This section aims to define some terms that pop up throughout the user guide. Th
 good to take note of them before continuing.
 
 ### Command-specific terms
-1. **Command**: A piece of instruction that a user types into the command input box to instruct Cobb's behaviour.
-   Commands have many variations, and can serve many purposes.
-2. **Syntax**: The manner in which is a command is to be typed. Can be thought of as the "format" of the command.
-3. **Parameter**: An input to a command. Can be thought of as a piece of information a user provides that determines the
+1. **Command String**: The text you input into the [command input box](#1-command-input-and-output-boxes), which is
+translated to a Command by Cobb
+2. **Command**: An instruction that is executed by Cobb and translated from your Command String 
+3. **Syntax**: The manner in which is a Command String is to be typed. Can be thought of as the "format" of the command string.
```

Comment:
```suggestion
3. **Syntax**: The manner in which a Command String is to be typed. Can be thought of as the "format" of the command string.
```

Reply:
I think the original sentence makes more grammatical sense haha